pytest/tests/mocknet/helpers/load_test_utils.py

```diff
@@ -0,0 +1,239 @@
+
+
+def send_random_transactions(node_account,
+                             test_accounts,
+                             max_tps_per_node,
+                             rpc_infos=None):
+    logger.info("===========================================")
+    logger.info("New iteration of 'send_random_transactions'")
+    if rpc_infos:
+        addr = random.choice(rpc_infos)[0]
+    else:
+        addr = None
+    status = mocknet_helpers.get_status(addr=addr)
+    base_block_hash = base58.b58decode(
+        status['sync_info']['latest_block_hash'].encode('utf-8'))
+    base_block_height = status['sync_info']['latest_block_height']
+    pmap(
+        lambda index_and_account: random_transaction(
+            index_and_account[1],
+            index_and_account[0],
+            node_account,
+            test_accounts,
+            max_tps_per_node,
+            base_block_hash=base_block_hash,
+            base_block_height=base_block_height), enumerate(test_accounts))
+
+
+def init_ft(node_account):
+    tx_res = node_account.send_deploy_contract_tx(
+        '/home/ubuntu/fungible_token.wasm')
+    logger.info(f'ft deployment {tx_res}')
+    mocknet_helpers.wait_at_least_one_block()
+
+    s = f'{{"owner_id": "{node_account.key.account_id}", "total_supply": "{10**33}"}}'
+    tx_res = node_account.send_call_contract_raw_tx(node_account.key.account_id,
+                                                    'new_default_meta',
+                                                    s.encode('utf-8'), 0)
+    logger.info(f'ft new_default_meta {tx_res}')
+
+
+def init_ft_account(node_account, account):
+    s = f'{{"account_id": "{account.key.account_id}"}}'
+    tx_res = account.send_call_contract_raw_tx(node_account.key.account_id,
+                                               'storage_deposit',
+                                               s.encode('utf-8'),
+                                               (10**24) // 800)
+    logger.info(f'Account {account.key.account_id} storage_deposit {tx_res}')
+
+    # The next transaction depends on the previous transaction succeeded.
+    # Sleeping for 1 second is the poor man's solution for waiting for that transaction to succeed.
+    # This works because the contracts are being deployed slow enough to keep block production above 1 bps.
+    mocknet_helpers.wait_at_least_one_block()
+
+    s = f'{{"receiver_id": "{account.key.account_id}", "amount": "{10**18}"}}'
+    logger.info(
+        f'Calling function "ft_transfer" with arguments {s} on account {account.key.account_id}'
+    )
+    tx_res = node_account.send_call_contract_raw_tx(node_account.key.account_id,
+                                                    'ft_transfer',
+                                                    s.encode('utf-8'), 1)
+    logger.info(
+        f'{node_account.key.account_id} ft_transfer to {account.key.account_id} {tx_res}'
+    )
+
+
+def init_function_call_state(test_accounts):
+    global function_call_state
```

Comment:
Does this need to be global? Why not pass it as an argument to the functions that use it? Maybe there could be something like

```python
class TestState:
  def __init__(self, test_accounts):
    ...
```

and then instead of passing the `test_accounts` list everywhere you could pass that object that keeps track of that list and this `function_call_state` variable. (or if that is too much, could just pass it as an argument normally)